test/test_server.js

```diff
@@ -34,116 +40,33 @@ function TestServer(config, printLogs) {
   this.mailbox = mailbox(config.smtp.api.host, config.smtp.api.port, this.printLogs)
 }
 
-function waitLoop(testServer, url, cb) {
-  request(
```

Comment:
Switching to a single process increased the speed by 2x, (from 4min to 2), and then interestingly, killing this wait loop and just waiting on the promise to resolve from `bin/key_server` got us to 1 minute!